src/NServiceBus.AcceptanceTesting/AcceptanceTestingContainer/AcceptanceTestingContainer.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace NServiceBus
+{
+    using Container;
+    using ObjectBuilder.Common;
+    using Settings;
+
+    public class AcceptanceTestingContainer : ContainerDefinition
```

Comment:
should we add some comments describing the purpose of this adapter container?

Reply:
Not sure what to say other than "A container for acceptance testing purposes" ? 

Or are you getting at the "The container is immutable after first resolve" behaviour?

Reply:
yes, documenting the immutability behavior and that we check it's essentially part of our container seam API.

Reply:
Added, see if it makes sense?